sql/glean_clients_daily_v1.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TEMP FUNCTION
+  udf_geo_struct(country STRING,
+    city STRING,
+    geo_subdivision1 STRING,
+    geo_subdivision2 STRING) AS ( IF(country IS NULL
+      OR country = '??',
+      NULL,
+      STRUCT(country,
+        NULLIF(city,
+          '??') AS city,
+        NULLIF(geo_subdivision1,
+          '??') AS geo_subdivision1,
+        NULLIF(geo_subdivision2,
+          '??') AS geo_subdivision2)));
+CREATE TEMP FUNCTION
+  udf_json_mode_last(list ANY TYPE) AS ((
+    SELECT
+      ANY_VALUE(_value)
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS _offset
+    GROUP BY
+      TO_JSON_STRING(_value)
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1));
+CREATE TEMP FUNCTION
+  udf_mode_last(list ANY TYPE) AS ((
+    SELECT
+      _value
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS
+    _offset
+    GROUP BY
+      _value
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1 ));
+--
+WITH
+  base AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    LOWER(client_info.client_id) AS client_id,
+    *
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix.metrics_v1`
```

Comment:
Only using the metrics ping for now, since that seems to be the only place where some of the items we want (default_browser, default_search) live. Do you expect we're leaving out users that should be counted by not considering other ping types, @fbertsch ?

Reply:
It is worth verifying this with a query. A metrics ping is not sent if a user does not report any metrics in a session.

Reply:
Based on reading the glean docs and some discussion in `#glean` yesterday, I'm going to move to looking at both baseline and metrics pings.